in/command.go

```diff
@@ -80,59 +80,60 @@ func (command *Command) Run(destinationDir string, request Request) (Response, e
 		isInitialVersion = request.Source.InitialVersion != "" && request.Version.VersionID == request.Source.InitialVersion
 	}
 
-	if isInitialVersion {
-		if request.Source.InitialContentText != "" || request.Source.InitialContentBinary == "" {
-			err = command.createInitialFile(destinationDir, path.Base(remotePath), []byte(request.Source.InitialContentText))
-			if err != nil {
-				return Response{}, err
+	if !request.Source.SkipDownload {
```

Comment:
I don't like the deep nesting this resulted in. Maybe just short-circuit in the `downloadFile` method instead?

Reply:
good point, we could eather just exclude the `downloadFile` and the `Unpack` part or modify the `downloadFile` method and update the `if request.Params.Unpack {` condition.

Update the `downloadFile` method could works but it feel strange to have a download method and put a parameter to not download.

WDYT ?

Reply:
True, we can just push the conditional down to wrap the download and unpack part instead of making the download method conditional.